lib/tasks/evm_application.rb

```diff
@@ -110,4 +110,13 @@ def self.update_stop
     File.write(tempfile, " ")
     stop
   end
+
+  def self.set_region_file(region_file, new_region)
+    old_region = region_file.exist? ? region_file.read.to_i : nil
+
+    return if new_region == old_region
+
+    _log.info("Changing REGION file from [#{old_region}] to [#{new_region}]. Restart to use the new region.")
+    region_file.write(new_region)
```

Comment:
Yea, if you don't want a region file, just setting env `REGION` when migrating the database (for the first time).
But there was reluctance to removing the `REGION` file, so I stopped pushing this agenda.